R/Task.R

```diff
@@ -257,17 +257,14 @@ Task = R6Class("Task",
 
     #' @description
     #' Returns a slice of the data from the [DataBackend] as a `data.table`.
-    #' Rows default to observations with role `"use"`, and
-    #' columns default to features with roles `"target"` or `"feature"`.
-    #' If `rows` or `cols` are specified which do not exist in the [DataBackend],
-    #' an exception is raised.
+    #' Rows default to observations with role `"use"`, and columns default to features with roles `"target"` or `"feature"`.
+    #' Rows must be a subset of `$row_roles$use`.
```

Comment:
```suggestion
    #' Rows must be a subset of `$row_ids`.
```

Reply:
I think we should start to maybe even slowly move away from the `row_roles` and maybe only use the `$row_ids` anymore, as all other roles are removed anyway.